src/malli/experimental/time/generator.cljc

```diff
@@ -11,7 +11,7 @@
 #?(:clj (set! *warn-on-reflection* true))
 
 (def zone-id-gen
-  (mg/generator (into [:enum] (map #(. ZoneId of ^String %)) (. ZoneId getAvailableZoneIds))))
+  (mg/generator (into [(:enum (m/base-schemas))] (map #(. ZoneId of ^String %)) (. ZoneId getAvailableZoneIds))))
```

Comment:
`m/base-schemas` pull a lot of schemas in, you can use instead:

```clojure
(into [(m/-enum-schema)] ..children...
```

or directly:

```clojure
(m/int-schema (m/-enum-schema) nil ...children...
```

the latter being faster.

Reply:
very cool! thanks, today I learned...